docker-compose-dev.yml

```diff
@@ -0,0 +1,56 @@
+# dev compose
+# extend from docker-compose-common.yml
+geocoder:
+  extends:
+    file: docker-compose-common.yml
+    service: geocoder
+  links:
```

Comment:
That's dumb.  I'm honestly not a big fan of how they implemented their "extends".  You end up with a lot of boilerplate copy/paste.  Ah well.  Still better than raw `docker run` for dev.